src/main/webapp/player/simid_player.html

```diff
@@ -120,13 +120,14 @@
       <tr>
         <td>Interactive Creative File</td>
         <td><input type="text" id="creative_url" size = 30 
-           value="../creatives/simid_overlay.html">
+           value="https://step-capstone-team33-2020.appspot.com/creatives/maps/simid_map.html">
```

Comment:
Hopefully, this could save us some repetitive copy-pasting for now. This might have to change back when we put our creative on the github page.